calculator.py

```diff
@@ -0,0 +1,99 @@
+from enum import IntEnum
+from dotenv import load_dotenv
+import os
+
+
+def addition(x, y):
+    return x + y
+
+
+def subtraction(x, y):
+    return x - y
+
+
+def multiplication(x, y):
+    return x * y
+
+
+def division(x, y):
+    if y == 0:
+        return "Error - division by 0"
+    else:
+        return x / y
+
+
+def exponentiation(x, y):
+    if y == 0:
+        return 1
+    else:
+        return x ** y
+
+
+def remainder(x, y):
+    if y == 0:
+        return "Error - division by 0"
+    else:
+        return x % y
+
+
+def get_name():
+    load_dotenv()
+    name = os.getenv('name1')
```

Comment:
Czemu name1? 

Dodatkowo, w .env stosujemy pisownie zawsze wielkimi literami z snake case. Dodatkowo jeżeli nie wiesz co to, to poczytaj też o camel case. W przyszłości się przyda jakbyś z kimś gadała :D

Reply:
tak nazwałam zmienną w pliku, a co do snake i camel case - jutro doczytam i zmienię.